src/index.ts

```diff
@@ -143,8 +143,13 @@ export const htmlPlugin = (configuration: Configuration = { files: [], }): esbui
         const template = (htmlTemplate && fs.existsSync(htmlTemplate)
             ? await fs.promises.readFile(htmlTemplate)
             : htmlTemplate || '').toString()
-        const compiledTemplateFn = lodashTemplate(template || defaultHtmlTemplate)
-        return compiledTemplateFn({ define })
+
+        if (define === undefined) {
+            return template
```

Comment:
Maybe I'm wrong here, but I feel like this should be:

```suggestion
            return template || defaultHtmlTemplate
```

Reply:
You are right, thanks for checking!

Fixed slightly differently:
https://github.com/craftamap/esbuild-plugin-html/pull/46/commits/5fffc45ea7c68c7f150fb4e60683fe95271ad28d